dask_cuda/utils.py

```diff
@@ -56,10 +57,12 @@ def __init__(
         self.async_alloc = async_alloc
         self.logging = log_directory is not None
         self.log_directory = log_directory
+        self.rmm_track_allocations = track_allocations
 
     def setup(self, worker=None):
+        import rmm
```

Comment:
Are we safe to always import RMM here? If this function is always run as part of the `RMMSetup` plugin, wouldn't that make RMM an implicit dependency for starting a CUDA cluster?

Reply:
Great catch @charlesbluca , indeed I failed to realize that. Yes, we should remove it from the "main" context of `setup` and leave it solely within the local contexts that exist today, iff one of those options to enable RMM is set by the user.

Reply:
Thanks both! I've accepted @pentschev's suggestions here.